srcpkgs/video-compare/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'video-compare'
+pkgname=video-compare
+version=20250420
+revision=1
+build_style=gnu-makefile
+makedepends="ffmpeg6-devel SDL2-devel SDL2_ttf-devel"
+short_desc="Split screen video comparison tool using FFmpeg and SDL2"
+maintainer="sunshinehunter <sunshinehunterde@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/pixop/video-compare"
+changelog="https://github.com/pixop/video-compare/releases"
+distfiles="https://github.com/pixop/video-compare/archive/${version}.tar.gz"
```

Comment:
```
distfiles="https://github.com/pixop/video-compare/archive/refs/tags/${version}.tar.gz"
```
The archive archive link clashes with branches and possibly commits in this case since its just valid hex.